_includes/scripts.html

```diff
@@ -66,7 +66,7 @@ <h5>Popular tags: </h5>
           <% } %>
           href="#/page/<%=pageNumber%>/tags/<%-encodeURIComponent(tagsString)%>"><%=pageNumber%></a>
       <% }}) %>
-      <a id="next" href="#/page/<%=currentPageNumber-(-1)%>/tags/<%-encodeURIComponent(tagsString)%>" <% if(currentPageNumber >= lastPage) { %> class="not-active" <% } %>>Next &raquo;</a>
+      <a id="next" href="#/page/<%=parseInt(currentPageNumber)+1%>/tags/<%-encodeURIComponent(tagsString)%>" <% if(currentPageNumber >= lastPage) { %> class="not-active" <% } %>>Next &raquo;</a>
```

Comment:
How about parsing `currentPageNumber` in `renderProjects`, so the template can be simpler? We allow number coersion above in the interval calculation, too; ideally we would replace anything that's not a positive page number with 1 before it reaches the template.